components/binary_sensor/mpr121.rst

```diff
@@ -41,12 +45,16 @@ Base Configuration:
 
 - **address** (*Optional*, integer): The i2c address of the sensor. Defaults to ``0x5A``.
 - **id** (*Optional*, :ref:`config-id`): Set the ID of this sensor.
+- **touch_debounce** (*Optional*, integer): The number of bounces before a touch is recognized. The value should be between 0 and 7.
+- **release_debounce** (*Optional*, integer): The number of bounces before a release is recognized. The value should be between 0 and 7.
 
 Binary Sensor Configuration:
 
 - **name** (**Optional**, string): The name for the binary sensor.
 - **id** (*Optional*, :ref:`config-id`): Manually specify the ID used for code generation.
 - **channel** (*Required*, integer): The channel number at the mpr121 the touchkey is connected to.
+- **touch_threshold** (*Optional*, integer): Typically the touch threshold is a little bigger than the release threshold. Value should be between 5 and 30.
```

Comment:
Before explaining what range this value should be in, explain what this value/threshold means. I don't know what they mean from this description.

Also add default value.